stylesheets/components/rating.css

```diff
@@ -0,0 +1,16 @@
+@import url(/Stylesheets/main.css);
+
+.check {
+  margin-top: 1rem;
+  position: relative;
+  top: 5rem;
+  flex-direction: row;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
You can make separate class for display flex and center in utility class and directly use

Reply:
yes you are right but i have only used one time here , so  i didnt take it seperate .